packages/docsearch-react/src/DocSearchModal.tsx

```diff
@@ -27,6 +27,13 @@ export interface DocSearchModalProps extends DocSearchProps {
   onClose?(): void;
 }
 
+const setRealHeight = () => {
+  // We update the value of the CSS variables
+  const vh = window.innerHeight * 0.01;
+  const modalElement = document.querySelector<HTMLElement>('.DocSearch-Modal')!;
+  modalElement.style.setProperty('--vh', `${vh}px`);
```

Comment:
How does that play out if other code on people's website use a `vh` CSS property? Should we scope its naming to DocSearch? (e.g., `--docsearch-vh`)